lib/filters/mongodb_version_filter.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+var semver = require('semver'),
+    f = require('util').format;
+
+var MongoDBVersionFilter = function(options) {
+  options = options || {};
+  // Get environmental variables that are known
+  var version = null;
+
+  this.beforeStart = function(object, callback) {
```

Comment:
Add to `applyFilters` in metamocha to allow for filters to have `beforeStart` functions, that take the given runtime configuration and run before the `filter` functions are applied